.gitignore

```diff
@@ -12,6 +12,7 @@ dist
 !.npmrc
 !.travis.yml
 
+yarn.lock
```

Comment:
why?

Reply:
I am using yarn, `package-lock.json` was already ignored (i don't know why), so I decided that the right decision would be ignoring `yarn.lock` until we know what's going on.

Reply:
Here is a short summary of why the file is not commited that aligns with my line of thought: https://github.com/sindresorhus/ama/issues/479#issuecomment-310661514

Thanks for adding `yarn.lock` to the list!